chapters/shaders.adoc

```diff
@@ -2498,6 +2498,86 @@ In other shader stages, each invocation in a subgroup must: be in the same
 
 Only <<limits-subgroup-supportedStages, shader stages that support subgroup
 operations>> have defined subgroups.
+
+[NOTE]
+.Note
+====
+In shaders, there are two kinds of uniformity that are of primary interest to
+applications: subgroup uniform and dynamically uniform.
+
+While dynamically uniform appears to imply subgroup uniform,
```

Comment:
nit: "appears to" reads as if the spec implies this. Maybe something like: "A common assumption is that dynamically uniform implies .." or "One could easily make the assumption that..".

Reply:
Reworded.